### DISCUSSION TITLE
Pixel Experience Thirteen Clean Build Required

### DISCUSSION THREAD

**Discussion by ImSurajx** [ROM-builders/temporary#16643] [Show and tell]
https://cirrus-ci.com/build/4784612493230080

**Comment by sanjeevstunner** [comment]
> https://cirrus-ci.com/build/4784612493230080

https://github.com/ROM-builders/temporary/discussions/16643#discussion-4717037

Add this to your manifest, and you're good

**Reply by ImSurajx** [reply]
What Bro Send Text :) Please
